src/index.html

```diff
@@ -117,7 +117,7 @@ <h2 class="title__heading">
         </div>
         <ul class="our-services__list">
           <li class="our-services__service service">
-            <img src="#" alt="Research" class="service__icon">
+            <img src="images/smartphone.png" alt="Research" class="service__icon">
```

Comment:
The some thing as in the first comment, but here you did it better. My way to store images:
`images/our-services/smartphone.png`
For this web page it may be too much, but when you have a bunch of images it makes easier to manipulate with them.